model/map/src/main/java/org/keycloak/models/map/user/MapUserProvider.java

```diff
@@ -93,6 +94,10 @@ public MapUserProvider(KeycloakSession session, MapStorage<MapUserEntity, UserMo
         session.getTransactionManager().enlist(tx);
     }
 
+    private Boolean getUsernameCaseSensitiveAttribute(RealmModel realm) {
+        return realm.getAttribute(USERNAME_CASE_SENSITIVE, Boolean.FALSE);
```

Comment:
The default value (`Boolean.FALSE`) should also be a constant in `MapUserProviderFactory`.